app/controllers/print_jobs_controller.rb

```diff
@@ -28,7 +28,7 @@ def print_job_params
         # While we COULD carefully define the current label structure, we gain nothing by doing so and make
         # future changes more painful.
         permitted[:labels] = params.require(:print_job)[:labels].map(&:permit!)
-        permitted[:labels_sprint] = params.require(:print_job)[:labels_sprint][:sprint].map(&:permit!)
```

Comment:
This is what caused the issue. It only permitted `sprint` attributes, but not other attributes under `labels_sprint`. 

We should increase test coverage for additional attributes and qc definitions later on.